docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -38,15 +38,16 @@ You can use a short name - *browser alias* - to identify these browsers when lau
 
 The following table lists browsers that can be detected automatically.
 
-Browser           | Browser Alias
------------------ | -------------------
-Chromium          | `chromium`
-Google Chrome     | `chrome`
-Internet Explorer | `ie`
-Microsoft Edge    | `edge`
-Mozilla Firefox   | `firefox`
-Opera             | `opera`
-Safari            | `safari`
+Browser                                      | Browser Alias
+---------------------------------------------| -------------------
+Chromium                                     | `chromium`
+Google Chrome (Stable, Beta, Dev channel)    | `chrome`
```

Comment:
I suppose so detailed info is unnecessary (`(Stable, Beta, Dev channel)`). What do you think @VasilyStrelyaev